contracts/axelar-gateway/src/test.rs

```diff
@@ -276,3 +276,65 @@ fn rotate_signers() {
         (source_chain, source_address),
     );
 }
+
+#[test]
+fn transfer_operatorship() {
+    let (env, contract_id, client) = setup_env();
+    let operator = Address::generate(&env);
+    let new_operator = Address::generate(&env);
+
+    initialize(&env, &client, operator.clone(), 1, randint(1, 10));
+
+    assert_eq!(client.operator(), operator);
+
+    client
+        .mock_auths(&[MockAuth {
+            address: &operator,
+            invoke: &MockAuthInvoke {
+                contract: &contract_id,
+                fn_name: "transfer_operatorship",
+                args: (&new_operator,).into_val(&env),
+                sub_invokes: &[],
+            },
+        }])
+        .transfer_operatorship(&new_operator);
+
+    assert_emitted_event(
```

Comment:
can you create a `assert_last_emitted_event` helper? don't need to pass a weird looking `-1` everytime

Reply:
Agreed. created a `assert_last_emitted_event ` helper and updated all related unit tests.